deploy/cr.yaml

```diff
@@ -36,10 +36,10 @@ spec:
     size: 3
     image: percona/percona-xtradb-cluster:8.0.23-14.1
     autoRecovery: true
-#    expose:
-#      enabled: true
-#      type: LoadBalancer
-#      trafficPolicy: Local
+#     expose:
+#       enabled: true
+#       type: LoadBalancer
+#       trafficPolicy: Cluster
```

Comment:
why are we changing the `trafficPolicy`?

Reply:
reverted